simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/ThreadExports.java

```diff
@@ -32,6 +32,7 @@
  */
 public class ThreadExports extends Collector {
   private final ThreadMXBean threadBean;
+  private boolean includeDeadlockedThreads = true;
```

Comment:
This needs locking.

Reply:
Locking in what way? Better to set it in overloaded constructor?

Reply:
That's one option, this can be accessed and mutated concurrently which is the issue. You could also split this out to a completely separate collector.